packages/manager/modules/pci/src/projects/project/private-registry/api-url/api-url.component.js

```diff
@@ -2,8 +2,11 @@ import controller from './api-url.controller';
 import template from './api-url.html';
 
 const component = {
-  template,
+  bindings: {
+    trackClick: '<',
```

Comment:
```suggestion
    onClick: '<',
```
we could pass another callback than tracking

Reply:
But i think this be confusing. Its more generic and could be anything.